### DISCUSSION TITLE
Container images can't be found

### DISCUSSION THREAD

**Discussion by orzklv** [community/community#127227] [Packages]
### Select Topic Area

Bug

### Body

After deleting a few repositories that was linked with Github hosted docker containers, packages just became unavailable but you can see them in my profile's package list. 

For example:
![image](https://github.com/community/community/assets/54666588/8f2cb22a-1438-422b-af6f-ccfb7d9b3f73)
This is the list of container packages shown on my profile

![image](https://github.com/community/community/assets/54666588/eb053f1c-1f22-42ef-a933-e90f233d16cc)
But if i open those that were linked with deleted repo, it just shows 404. 

I would clean unavailable packages and remove them from my profile at least, so I may continue with newer ones.

**Comment by ebndev** [comment]
Hi @orzklv, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our Packages category, as this topic might be more relevant there! Hopefully that might help someone give you a nudge in the right direction.
